nnpackage/schema/circle_schema.fbs

```diff
@@ -74,7 +74,7 @@ enum TensorType : byte {
   // Belows are using negative value to represent not existing TensorType on TensorFlow Lite schema
 
   UINT4 = -1,
-  // GGML_Q{X}_{Y} are follow ggml quantization spec (https://github.com/ggerganov/ggml)
+  // GGML_Q{X}_{Y} conform to the GGML (https://github.com/ggerganov/ggml) quantization specification.
```

Comment:
```suggestion
  // GGML_Q{X}_{Y} follow the GGML quantization spec (https://github.com/ggerganov/ggml).
```

If you don't prefer, I would like to minimize the change.

- removes `are`
- changes `ggml` to `GGML`
- add `the`

Before your comment, I moved the url (https://github.com/ggerganov/ggml) immediately after `GGML` since I cannot find the GGML quantization specification under the url. Practically, it is just repository which has header and implementaiton of GGML.

However, I restored it back since it may change something you want to keep.

Reply:
LGTM